tck/src/main/java/ee/jakarta/tck/concurrent/spec/ContextService/contextPropagate/ContextServiceDefinitionServlet.java

```diff
@@ -60,8 +60,8 @@
                           cleared = StringContext.NAME,
                           unchanged = TRANSACTION)
 @ContextServiceDefinition(name = "java:module/concurrent/ContextB",
-                          cleared = TRANSACTION,
-                          unchanged = { APPLICATION, IntContext.NAME },
+                          cleared = {TRANSACTION, APPLICATION},
+                          unchanged = { IntContext.NAME },
```

Comment:
Making this change would only correct the assertion on APPLICATION context and not the subsequent assertion on IntContext.  Also, is it safe with respect to all of the other TCK tests that use "java:module/concurrent/ContextB" to alter this configuration?  It seems like the safest change would be to either adjust the assertions to match the specified annotation values, or otherwise create an additional ContextServiceDefinition with values that the assertions expect.